xml/ssh_banners.xml

```diff
@@ -304,6 +304,16 @@ fingerprint SSH servers.
     <param pos="0" name="os.product" value="Linux"/>
     <param pos="0" name="os.version" value="13.04"/>
   </fingerprint>
+  <fingerprint pattern="^OpenSSH_(\d+\.\d+(?:\.\d+)?(?:p\d+)?)[_|-](hpn13v11)$">
```

Comment:
During my final review of this, I noticed that the `hpn` part here is hardcoded.  It looks like this can absolutely be something more than just hpn13v11, for example I see what looks to be {{hpn14v5}} and {{hpn14v9}} at https://launchpad.net/~yoda-jazz-kc/+archive/ubuntu/hpn-ssh.  I'd suggest just changing the relevant part of this regex to `(hpn\d+v\d+)`